stable/coredns/templates/secret.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.kubernetesTls.enabled }}
```

Comment:
I'd like to see an option for the user to specify a secret they've already created, that way they can elect to not store the secrets in helm values.